tests/unit/test_conf.py

```diff
@@ -0,0 +1,46 @@
+#pylint: skip-file
+import os
+import unittest
+
+import yaml
+
+
+class TestAtomicUnit(unittest.TestCase):
+    """
+    Checks that the configuration file provided is valid.
+    """
+
+    def setUp(self):
+        """
+        Provide the conf file contents freshly for each test.
+        """
+        self.conf = open(os.path.sep.join(['..', '..', 'atomic.conf']), 'r')
+
+    def tearDown(self):
+        """
+        Close the config file after ever test.
+        """
+        self.conf.close()
+
+    def test_config_file_is_valid_yaml(self):
+        """
+        Verifies atomic.conf is valid YAML.
+        """
+        self.assertEquals(type(yaml.safe_load(self.conf)), dict)
+
+    def test_config_file_is_valid_yaml_with_items_uncommented(self):
+        """
+        Verifies atomic.conf is valid YAML when examples are uncommented.
+        """
+        # If a command line has a space after it, it's a comment
+        # If a comment line has no space after it, it's an example
+        uncommented = []
+        for line in self.conf.readlines():
+            if line.startswith('#') and len(line) > 2 and line[1] != ' ':
```

Comment:
Maybe not enough tea yet today, but should it be checking "> 1" rather than "> 2"?   Otherwise wouldn't '#A' get skipped?